.github/dependabot.yml

```diff
@@ -8,6 +8,8 @@ updates:
     # Check for updates once a week
     schedule:
       interval: "weekly"
+      day: "sunday"
+      time: "22:00"
```

Comment:
Why set a specific time for these actions?

Reply:
This can help with an expected review time-window, random than a random time for those PRs

Reply:
Okay, understood and sounds good. Note that in our experience with `interval: "weekly"` for the past year or so, then dependabot PRs have reliably come on Sunday evening.